x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/EvaluatorImplementer.java

```diff
@@ -544,6 +606,7 @@ public void buildInvocation(StringBuilder pattern, List<Object> args, boolean bl
                 return;
             }
             if (blockStyle) {
+                // TODO: Support multiValued for non-BytesRefBlock types
```

Comment:
I'd throw for now until you need it. That way the compiler will fail and folks can get an error message you wrote.

Reply:
Added code path for multi-value case.